cpu/stm32_common/periph/rtt.c

```diff
@@ -67,6 +67,23 @@
 #endif
 #endif
 
+#if defined(CPU_FAM_STM32L4)
+#define IMR_REG             IMR2
+#define EXTI_IMR_BIT        EXTI_IMR2_IM32
+#elif defined(CPU_FAM_STM32L0)
+#define IMR_REG             IMR
+#define EXTI_IMR_BIT        EXTI_IMR_IM29
+#else
+#define IMR_REG             IMR
+#define FTSR_REG            FTSR
+#define RTSR_REG            RTSR
+#define PR_REG              PR
+#define EXTI_FTSR_BIT       EXTI_FTSR_TR23
+#define EXTI_RTSR_BIT       EXTI_RTSR_TR23
+#define EXTI_IMR_BIT        EXTI_IMR_MR23
+#define EXTI_PR_BIT         EXTI_PR_PR23
```

Comment:
Should be `XX17` for `stm32f1`, see #13013.

Reply:
Good to know but I'm realizing that stm32f1 is out of the scope of this PR since it provides its own implementation of rtt.